### PR TITLE
The working Pick up controller

### PR DESCRIPTION
**Pull request by Daniel-Frederick**
Interact/Pick up: F
Drop Object: R
Throw: Left Mouse

### CODE REVIEW

**Comment by zoogies** [comment]
To me drop would make more sense on something like G, but thats really just a nitpick

File: game/Assets/Scripts/Outliner.cs [hunk 1]
```diff
@@ -3,8 +3,13 @@
 public class Outliner : MonoBehaviour
 {
     [SerializeField] private Interactor interactor;
+    [SerializeField] private Camera playerCamera;  // Reference to the main camera
     private Transform highlightedObject;
 
+    private void Start()
+    {
+    }
+
```
Comment (zoogies) [inline game/Assets/Scripts/Outliner.cs:12]: Maybe remove if unused

**Comment by Daniel-Frederick** [comment]
I could change that in 5 seconds if needed

File: game/Assets/Scripts/Outliner.cs [hunk 2]
```diff
@@ -13,15 +18,15 @@ void Update()
...
         }
 
-        // debug: visualize ray
-        // BUG: ray comes from the players feet, not the camera (with angle) - TODO
-        // Debug.DrawRay(interactor.transform.position, interactor.transform.forward * interactor.InteractRange, Color.red);
+        // Create ray from camera instead of player position
+        Ray ray = new Ray(playerCamera.transform.position, playerCamera.transform.forward);
+
+        // Debug visualization (optional)
+        Debug.DrawRay(playerCamera.transform.position, playerCamera.transform.forward * interactor.InteractRange, Color.red);
```
Comment (zoogies) [inline game/Assets/Scripts/Outliner.cs:25]: Should this be commented out for prod?

**Review by zoogies — CHANGES_REQUESTED** [review]
Overall it looks good, nice work!!!! Let me know your thoughts on these comments though and then we can merge depending

**Comment by zoogies** [comment]
G has been my experience by default for many games, but I say we just leave it on R

**Comment by Daniel-Frederick** [comment]
> to nitpick more g is a bad drop button and makes you extend your finger too far

I agree

@zoogies I will remove those for sure, thanks

**Comment by Cavekitty16** [comment]
I love how easily persuaded yall are

**Review by zoogies — APPROVED** [review]
LGTM, thanks

**Pull request merged**